### PR TITLE
[FUSEQE-5668] Add dynamoDb basic test set

### PR DESCRIPTION
**Pull request by tplevko**
//test: `@dynamodb`

One of the tests should fail because (@dynamodb-query) because of [ENTESB-11767](https://issues.jboss.org/browse/ENTESB-11767)

### CODE REVIEW

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by avano** [comment]
retest this please

**Comment by avano** [comment]
@tplevko are you sure it should be backported to 1.7.x?

**Comment by syndesisqe** [comment]
Checkstyle verification failed with:
```
utilities/src/main/java/io/syndesis/qe/bdd/validation/
  DynamoDbValidationSteps.java:23: 'method call' child has incorrect indentation level 8, expected level should be 12. [Indentation]
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
Dynamodb connector | Webhook to dynamoDb - query item  - known issue
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by tplevko** [comment]
@mkralik3 can you please have a look? The one failure is due to ENTESB-11767

**Comment by mkralik3** [comment]
Yes

File: ui-tests/src/test/resources/features/integrations/dynamodb.feature [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+@dynamodb
+@integrations-dynamodb
+Feature: Dynamodb connector
+
+  Background: Clean application state
+    Given clean application state
+    Given log into the Syndesis
+    And delete dynamoDb DB table
+    And create new dynamoDb table with primary key "email" and sort key "company"
+     And created connections
```
Comment (mkralik3) [inline ui-tests/src/test/resources/features/integrations/dynamodb.feature]: extra empty space

File: ui-tests/src/test/resources/features/integrations/dynamodb.feature [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+
+@ui
+@datamapper
+@dynamodb
+@integrations-dynamodb
+Feature: Dynamodb connector
+
+  Background: Clean application state
+    Given clean application state
+    Given log into the Syndesis
```
Comment (mkralik3) [inline ui-tests/src/test/resources/features/integrations/dynamodb.feature]: Two Given steps. The second one can be And as well for more readability.

File: ui-tests/src/test/resources/features/integrations/dynamodb.feature (hunk 1 above)
Comment (tplevko) [reply ui-tests/src/test/resources/features/integrations/dynamodb.feature]: should be OK now

File: ui-tests/src/test/resources/features/integrations/dynamodb.feature (hunk 2 above)
Comment (tplevko) [reply ui-tests/src/test/resources/features/integrations/dynamodb.feature]: should be OK now

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Pull request merged**